layers/colors/packages.el

```diff
@@ -23,6 +24,21 @@
 ;;     :disabled t
 ;;     :init (add-hook 'emacs-lisp-mode-hook 'rainbow-blocks-mode)))
 
+(defun colors/init-color-identifiers-mode ()
+  (use-package color-identifiers-mode
+    :commands color-identifiers-mode
+    :defer t
+    :init
+    (progn
+      (spacemacs|diminish color-identifiers-mode "C " "C ")
```

Comment:
The first C has got to be Ⓒ (compare https://github.com/syl20bnr/spacemacs/blob/d993a021847cde2c42865bab6afa8adbb2edda0b/spacemacs/packages.el#L221)